contract/start-airdrop-plan.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
this is a product of `agoric run`; mixing the 2-bundle `agoric run` approach and the 1-bundle rollup approach is asking for trouble.

Reply:
these files were removed in latest commit.